transit_odp/publish/templates/publish/snippets/disruptions_map.html

```diff
@@ -0,0 +1,45 @@
+<head>
+  <style>
+    .mapboxgl-popup-content {
+      font: 400 15px/22px "GDS Transport", arial, sans-serif;
+      padding: 0;
+      width: 300px;
+    }
+
```

Comment:
Can we make the subheading on the pop up smaller and not bold as per the wireframes

Reply:
done